agent/util-scripts/pbench-move-results

```diff
@@ -135,7 +156,7 @@ if [[ -z "${rel}" ]]; then
     rel="unknown"
 fi
 # User-Agent HTTP header: <pbench-agent-ver-rel>:<FQDN>:<$USER>:<full path of this script>""
-user_agent="pbench-agent-${ver}-${rel}:${_pbench_full_hostname}:${USER}:${script_name}"
+user_agent="pbench-agent-${ver}-${rel}:${_pbench_full_hostname}:${USER}:${user}:${script_name}"
```

Comment:
What are we doing, here?  (An explanatory code comment would be useful....)  In particular, why is it important to include _both_ `$USER` and `$user`, without including `$controller` or `$prefix`?

(This change resulted in lots of churn in the gold files...is it really worth it?)